tests/unit/config/test_config.py

```diff
@@ -1531,6 +1645,47 @@ def test_rewrite_simple_posargs(self, tmpdir, newconfig):
         argv = conf.commands
         assert argv[0] == ["cmd1", "hello"]
 
+    def test_rewrite_posargs_pound(self, tmpdir, newconfig):
+        inisource = """
+            [testenv:py27]
+            args_are_paths = True
+            changedir = test#dir
+            commands = cmd1 {posargs:hello}
+        """
+        conf = newconfig([], inisource).envconfigs["py27"]
+        argv = conf.commands
+        assert argv[0] == ["cmd1", "hello"]
+
+        if sys.platform != "win32":
```

Comment:
Noting windows specific behaviour is happening here, cutting the dir at `#`.  May be a bug in test harness code, given that the next stanza is OK.